python_transport/wirepas_gateway/protocol/mqtt_wrapper.py

```diff
@@ -116,6 +123,13 @@ def _on_connect(self, client, userdata, flags, rc):
         if self.on_connect_cb is not None:
             self.on_connect_cb()
 
+    def _on_disconnect(self, userdata, rc):
+        if rc != 0:
+            self.logger.error(
+                "MQTT unexpected disconnection (network or broker originated)"
```

Comment:
Should  ...%s (%s)", connack_string(rc), rc) be added like in _on_connect.
Better to have all info available